src/convertToObject.js

```diff
@@ -7,6 +7,13 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  return sourceString.split(';').reduce((prev, next) => {
+    const [key, value] = next.split(':').map((item) => item.trim());
+
+    prev[key] = value;
```

Comment:
This line will throw an error if `key` is undefined. You must check if `key` is not empty before assigning it to the `prev` object to avoid adding undefined keys.